server/channels/app/web_broadcast_hooks.go

```diff
@@ -58,6 +63,9 @@ func (h *addFollowersBroadcastHook) Process(msg *platform.HookedWebSocketEvent,
 	if len(followers) > 0 && slices.Contains(followers, webConn.UserId) {
 		// Note that the client expects this field to be stringified
 		msg.Add("followers", model.ArrayToJSON([]string{webConn.UserId}))
+		if msg.Get("should_ack") != true {
```

Comment:
It does, if you have CRT on and you're following a thread, you will automatically get notifications unless you have the setting to not do that turned on.

Reply:
I don't think that's necessary the majority of posts, but for threads then yes we'll be ACKing a fair amount of those replies. Mind you it will only be limited to the number of people following those threads, and since I ended up getting pretty by having every user in every channel ACKing every post, I think we'll be okay with this.